frontend/src/main/pages/HomePage.jsx

```diff
@@ -2,6 +2,9 @@ import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
 import { useBackend } from "main/utils/useBackend";
 import DiningCommonsTable from "main/components/DiningCommons/DiningCommonsTable";
 
+import Form from "react-bootstrap/Form";
```

Comment:
This PR seems to now have changes to the home page and home page tests in it that are unrelated to the PR.

Can we clean this up?

This technique may help: it isn't specific to removing package-lock.json from a PR.   Just reset and then only commit the changes to the files that should be in this PR:

https://ucsb-cs156.github.io/topics/pull_requests/package_lock_json.html#the-fancy-git-way-with-git-reset---soft-